diddiparser2/lib/subprocessing.py

```diff
@@ -4,10 +4,19 @@
 
 import shlex
 import subprocess
+import sys
+import traceback
 
 from diddiparser2.exceptions import run_error
 
-DIDDISCRIPT_FUNCTIONS = "run_command"
+DIDDISCRIPT_FUNCTIONS = ("run_command", "run_python_code")
+
+# TODO: Find a better way to define PYTHON_GLOBALS.
+# By default, the globals are the same than
+# the globals(), but the locals are a bit
+# different.
+PYTHON_GLOBALS = globals()
+PYTHON_LOCALS = {"__name__": "__console__", "__doc__": None}

```

Comment:
I may have to fix this in the future. Having `subprocessing.PYTHON_GLOBALS` as the same value than the real `globals()` can cause unexpected issues.